gui/main.qml

```diff
@@ -5,7 +5,7 @@ Item
 {
     id: mainWindow
     width: 1024
-    height: 600
+    height: 720
```

Comment:
We should leave this at 600 for now at least since that is the resolution of the display we have for the prototype